Dockerfile

```diff
@@ -40,5 +40,5 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=${ARCH} \
 FROM gcr.io/distroless/static:latest
 WORKDIR /
 COPY --from=builder /workspace/manager .
-USER nobody
+# USER nobody
```

Comment:
it is removed as I was getting this error
```
E0910 19:34:39.892550       1 main.go:164] setup "msg"="problem running manager" "error"="listen tcp :443: bind: permission denied"  
```

Reply:
Let's undo this. We can control it using the manager Options struct, field=Port. We can create a flag for this in main.go.

Reply:
Filed this as https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/1120